fpga_interchange/examples/tests/const_wire/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_interchange_test(
+    name const_wire_basys3
+    device xc7a50t
+    package cpg236
+    tcl run.tcl
+    xdc wire.xdc
+    sources wire.v
+)
+
+add_interchange_test(
+    name const_wire_arty
+    device xc7a50t
+    package csg324
+    tcl run.tcl
+    xdc wire.xdc
```

Comment:
These will need different `XDC` because the pin definitions are different.

Reply:
fixed